frontend/src/routes/(authenticated)/myaccount/+page.ts

```diff
@@ -32,6 +33,9 @@ export async function _changeUserAccountData(input: ChangeUserAccountDataInput):
       //invalidates the graphql user cache, but who knows
       { additionalTypenames: ['Users'] },
     );
-  if (!result.error) void invalidate(`user:${input.userId}`);
+  if (!result.error) {
+    await refreshJwt();
+    await invalidate(`user:${input.userId}`);
```

Comment:
I believe we'll be able to get rid of the `{#if changed}` stanza at the bottom of `+page.svelte`, too:

```
{#if changed}
    <p class="text-warning">{$t('account_settings.notify')}</p>
{/if}
```

The invalidate re-runs load, which should automatically refresh the values on the page, and there should be no need to tell the user they need to sign out and sign back in.

Reply:
That will be gone when i submit a new pr